docs/android/emulator/device-manager.md

```diff
@@ -0,0 +1,434 @@
+
+  :::image type="content" source="media/device-manager/mac/26-reveal-in-finder-sml.png" alt-text="Results of pressing Reveal in Finder." lightbox="media/device-manager/mac/26-reveal-in-finder.png":::
+
+- **Factory Reset** &ndash; Resets the selected device to its default settings, erasing any user changes made to the internal state of the device while it was running (this also erases the current [Quick Boot](../deployment/debug-on-emulator.md#quick-boot) snapshot, if any). This change doesn't alter modifications that you make to the virtual device during creation and editing. A dialog box will appear with the reminder that this reset cannot be undone. Press **Factory Reset** to confirm the reset.
+
+  :::image type="content" source="media/device-manager/mac/27-factory-reset-m76.png" alt-text="Factory reset dialog.":::
+
+- **Delete** &ndash; Permanently deletes the selected virtual device. A dialog box will appear with the reminder that deleting a device cannot be undone. Press **Delete** if you are certain that you want to delete the device.
+
+  :::image type="content" source="media/device-manager/mac/28-delete-device-m76.png" alt-text="Delete device dialog.":::
+
+## Troubleshooting
+
+The following sections explain how to diagnose and work around problems that may occur when using the Android Device Manager to configure virtual devices.
+
+### Wrong version of Android SDK Tools
+
+TODO: Set the correct version numbers
+
+If Android SDK tools 26.1.1 or later isn't installed, you may see this error dialog on launch:
+
+:::image type="content" source="media/device-manager/mac/29-sdk-instance-error.png" alt-text="Android SDK instance error dialog.":::
+
+If you see this error dialog, press **OK** to open the Android SDK Manager. In the Android SDK Manager, go to the **Tools** tab and install the following packages:
+
+- **Android SDK Tools 26.1.1** or later
+- **Android SDK Platform-Tools 28.0.1** or later
+- **Android SDK Build-Tools 26.0.3** or later
+
+### Snapshot disables WiFi on Android Oreo
+
+If you've an AVD configured for Android Oreo with simulated Wi-Fi access, restarting the AVD after a snapshot may cause Wi-Fi access to become disabled.
+
+To work around this problem,
+
+01. Select the AVD in the Android Device Manager.
+
+01. From the **Additional Options** (&hellip;) menu, select **Reveal in Finder**.
+
+01. Navigate to **snapshots > default_boot**.
+
+01. Delete the **snapshot.pb** file:
+
+    :::image type="content" source="media/device-manager/mac/30-delete-snapshot-sml.png" alt-text="Location of the snapshot.pb file." lightbox="media/device-manager/mac/30-delete-snapshot.png":::
+
+01. Restart the AVD.
+
+After these changes are made, the AVD will restart in a state that allows Wi-Fi to work again.
+
+::: zone-end
+
+-->
+
+## Related links
```

Comment:
I'd delete any Related links sections, and instead just ensure the links are featured somewhere in the content. Related links is a hangover from when Xamarin docs weren't on docs.microsoft.com.